src/main/java/appeng/block/AEBaseBlock.java

```diff
@@ -150,8 +121,8 @@ public String toString()
 
 	public void registerNoIcons()
 	{
-		BlockRenderInfo info = this.getRendererInstance();
-		FlippableIcon i = new FlippableIcon( new MissingIcon( this ) );
+		final BlockRenderInfo info = this.getRendererInstance();
```

Comment:
That method is based on `this.getRenderer()` which returns an unparameterized `BaseBlockRenderer`. Least contract should be the base block and tile.